docker-compose.yml

```diff
@@ -1,16 +0,0 @@
-dev:
-  build: .
-  links:
-  - db
-  volumes:
-  - .:/src/graphql-sequelize
-  environment:
-    DB_DATABASE: graphql_sequelize_test
-    DB_USER: graphql_sequelize_test
-    DB_PASSWORD: graphql_sequelize_test
-
-db:
-  image: postgres:9.4
-  environment:
-    POSTGRES_USER: graphql_sequelize_test
-    POSTGRES_PASSWORD: graphql_sequelize_test
```

Comment:
I didn't remove it but moved to `test/support` folder. For MySQL, I duplicated the file to be specific to mysql. In package.json, we use the env variable `DB` for switching the `docker-compose` file and the project name.